src/cdk/v2/handler.ts

```diff
@@ -82,7 +84,6 @@ export async function processCdkV2Workflow(
   destType: string,
   parsedEvent: FixMe,
   feature: string,
-  logger: FixMe,
   requestMetadata: NonNullable<unknown> = {},
   bindings: Record<string, FixMe> = {},
 ) {
```

Comment:
should the logger be updated to ?
logger.debug(`Processing cdkV2 workflow`, {requestMetadata})

Reply:
Do we really need this log to have metadata information ? If yes, we can add it.

Reply:
Also this reqMetadata only contains
```javascript
{
      namespace: 'Unknown',
      cluster: 'Unknown',
      features: ctx.state?.features || {},
}
```
It is not the actual metadata object that we expect it to be.

Reply:
can pass {destType} - otherwise, there is no meaning to this log